load/UpdateDBPLPTSTable.py

```diff
@@ -18,7 +18,8 @@
 from UpdateDBPBibleTranslations import *
 from UpdateDBPLanguageTranslation import *
 
-GospelFilmStockNumberPrefix = "P2"
+DefaultLanguageId = 6414
+DefualtISO = "eng"
```

Comment:
rename to DefaultISO

Reply:
@bradflood , it has been fixed as you can view here: https://github.com/faithcomesbyhearing/dbp-etl/pull/196/files#diff-a8b84dcb2a008d41509d439d0dad3855517bc429e17563a012d78d868ced3024R22